music_album.rb

```diff
@@ -0,0 +1,14 @@
+require_relative 'item'
+
+class MusicAlbum < Item
+  attr_accessor :on_spotify
+
+  def initialize(publish_date, on_spotify)
+    super(publish_date)
+    @on_spotify = on_spotify
```

Comment:
- [ ] Kindly change this attribute to boolean. As per the requirements of the project, it should be a boolean
```suggestion
    @on_spotify = false
```